miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/PrometheusCollectionFilter.java

```diff
@@ -201,10 +203,35 @@ public void setStatus(int arg0) {
 
     @Override
     @CoverageIgnore
-    @SuppressWarnings("deprecation")
     public void setStatus(int arg0, String arg1) {
       status = arg0;
-      backingResponse.setStatus(arg0, arg1);
+      backingResponse.setStatus(arg0);
+    }
+
+    @Override
+    public void setContentLengthLong(long arg0) {
+      backingResponse.setContentLengthLong(arg0);
+
+    }
+
+    @Override
+    public String getHeader(String arg0) {
+      return backingResponse.getHeader(arg0);
+    }
+
+    @Override
+    public Collection<String> getHeaderNames() {
+      return backingResponse.getHeaderNames();
+    }
+
+    @Override
+    public Collection<String> getHeaders(String arg0) {
+      return backingResponse.getHeaders(arg0);
+    }
+
+    @Override
+    public int getStatus() {
+      return backingResponse.getStatus();
```

Comment:
All this nonsense existed because the original version of the servlet library didn't have `getStatus`. We can rip this out and modify the code below to just do `getStatus`. I can do that though.